src/styles/atoms/_accordion.scss

```diff
@@ -0,0 +1,70 @@
+@mixin dc-accordion {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: nowrap;
+}
+
+@mixin dc-accordion__item(
+    $accordion-content-selector: "dc-accordion__content"
+) {
+    display: block;
+    border-bottom: 1px solid $dc-gray70;
+    background-color: $dc-white;
+    padding-bottom: $dc-space100;
+
+    &:hover {
+        background-color: $dc-gray80;
+    }
+
+    .#{$accordion-content-selector} {
+        height: 0;
+        padding-right: $dc-space150;
+        padding-left: $dc-space150;
+        overflow: hidden;
+    }
```

Comment:
This is much simplified now, however, I think these three properties do not change in state open/closed so these can belong in it's own selector mixin named `dc-accordion__content`